test/pillar_test.exs

```diff
@@ -236,6 +236,39 @@ defmodule PillarTest do
                Pillar.query(conn, "SELECT * FROM #{table_name} LIMIT 1 FORMAT JSON")
     end
 
+    test "SimpleAggregateFunction(groupArrayArray, Array(Array(String)))", %{conn: conn} do
+      drop_table_sql = """
+      DROP TABLE IF EXISTS saf_table
+      """
+
+      create_table_sql = """
+      CREATE TABLE IF NOT EXISTS saf_table
```

Comment:
@hkrutzer It still falling into error, seems like problem in trailing comma

![image](https://user-images.githubusercontent.com/15660128/204490969-2c8a1caf-a99c-4486-95a3-8c9cc30fb8d8.png)

Reply:
It's on Clickhouse 22.1.3

Reply:
I've removed the trailing comma, let's see if this works and if not I will install 22.1.3 locally.

Reply:
It's worked)

I'm merging, thanks)